### PR TITLE
Hello "friendsofphp/proxy-manager-lts"

### PR DESCRIPTION
**Pull request by nicolas-grekas**
See https://github.com/Ocramius/ProxyManager/issues/630 for background.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
+# FriendsOfPHP / Proxy Manager LTS
 
-This package forks and replaces the [`ocramius/proxy-manager`](https://github.com/Ocramius/ProxyManager/) library.
-Unless they're caused by this fork, please report issues and submit new features to the origin library.
+This package is a fork of the excellent [`ocramius/proxy-manager`](https://github.com/Ocramius/ProxyManager/) library
+that adds long term support for a wider range of PHP versions.
 
-By using this package, you'll gain a versioning policy that won't force you to
-use the latest version of PHP *and* Composer to get bugfixes and new features.
+Unless they're caused by this very fork, please report issues and submit new features to the origin library.
```
Comment (alcaeus) [inline README.md]: I would add a note stating that while the library supports PHP 8.0, it does not support the full PHP 8 features:

```suggestion
Unless they're caused by this very fork, please report issues and submit new features to the origin library.

This fork runs on PHP 8 but does not support PHP 8 features (e.g. `mixed` type, union types) pending upstream implementation.
```

Comment (nicolas-grekas) [reply README.md]: I'd prefer not adding this as it will be hopefully wrong soon :)

Comment (alcaeus) [reply README.md]: 👍

**Review by alcaeus — APPROVED** [review]

**Review by wouterj — APPROVED** [review]

**Comment by kevinpapst** [comment]
When first I read about the package, I believed/hoped that it could be used as drop-in replacement for the original package (as previously stated in the readme with `forks and replaces`). The public discussions influenced the impression as well, that this was the intention of this package. 
Now with the namespace change this becomes impossible without problematic workarounds or maybe I missed something in that huge diff?

Probably worth mentioning in the README.md for all the people that hoped to get the proxy-manager version of `composer/package-versions-deprecated` 😁

**Comment by nicolas-grekas** [comment]
Thanks for asking @kevinpapst. That's the very question I'm asking myself right now. I also feel like it would be more convenient to "replace" the original package. I'm going to submit a separate PR with the appropriate patch so that we can discuss on actual code.

**Comment by kevinpapst** [comment]
Obviously just my personal view: keeping the namespace and being a replacement package solves the very real problem of all the people who are forced to stick to older versions or simply want to support older php versions in their libraries & apps for a longer time and/or who are using doctrine migrations and only have a transient dependency (probably a very large part of the user base).

Changing the namespace solves the issues only for a smaller subset of all users, while a bigger part of the community would be left out. 

You already agreed to take over that (ugly, but much appreciated) job here, so why making it even harder for yourself by directly binding the users to your namespace & package instead of ONLY solving the BC compatibility issue.

**Comment by nicolas-grekas** [comment]
Closing in favor of #2
Thanks for raising the point.

**Pull request closed without merging**